packages/nimble-components/src/wafer-map/modules/event-handler.ts

```diff
@@ -0,0 +1,49 @@
+import type { WaferMap } from '..';
+import type { WaferMapDie, EventHandlerData } from '../types';
+import { ZoomHandler } from './zoom-handler';
+
+/**
+ * EventHandler deals with user interactions and events
+ */
+export class EventHandler {
```

Comment:
What do you think of alternate names like `UserInteractionHandler` or `InteractionHandler`? "Event" is a pretty generic term which could refer to events that are from various sources so it would be nice to be more specific.

If you change this then we should update the corresponding type names too.

Reply:
I see your point, however I would still prefer to keep it as it is, since we might have indeed the need, in the future, to handle events coming from other sources as well, not just user interactions...

Reply:
I applied most of your feedback:
1. EventHandler was renamed to EventCoordinator.
2. Moved all of the event registrations within the EventCoordinator, except the resizeObserver which would require some more extensive refactoring - the resizeObserver is used to track the canvasSideLength, which in turn is used (among other things) to decide to re-render or not the canvas. Unless you think that there is no other way, I would prefer to keep the resizeObserver as it is